hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestHoodieClientMultiWriter.java

```diff
@@ -834,7 +834,7 @@ private HoodieWriteConfig buildWriteConfigForEarlyConflictDetect(String markerTy
           // the running interval of the async conflict detector so that the conflict can
           // be detected before the marker requests are processed at the timeline server
           // in the test.
-          .withMarkersTimelineServerBasedBatchIntervalMs(1000)
+          .withMarkersTimelineServerBasedBatchIntervalMs(10000)
```

Comment:
I run the test locally and find that we may call hasMarkerConflict before first detection is finished in AsyncTimelineServerBasedDetectionStrategy. So I increase the delay of schedule the marker creation request.